### PR TITLE
Milestone 8: Models

### PR DESCRIPTION
**Pull request by AbigiyaTY**
##  Responsive hamburger-menu

In this milestone I:

- Made hamburger menu responsive and connected nav item to their link in index.html
- Add styling to the responsive hamburger menu using CSS
- Add event listener to the hamburger menu using Javascript

### CODE REVIEW

**Review by vechicin — COMMENTED** [review]
# Hi @AbigiyaTY ✋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 😞 

## Invalid Code Review Request ⚠️ 

You have submitted a project, that is missing the ESlint linters. Please install them.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: css/style.css [hunk 1]
```diff
@@ -57,6 +58,60 @@ a {
...
+  left: 0;
+  z-index: 10;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: fixed;
+  background-color: #000;
+  transition: top 1s ease;
+  display: none;
+}
```
Comment (zieeco) [inline css/style.css]: - Please, kindly ensure your mobile menu and contents match the Figma design as much as you can below:

*your design*
n
* figma design*
match

Comment (zieeco) [reply css/style.css]: *README FILE*

> Please, kindly update your readme by doing the following to make your readme more professional.
- Add a "Getting started" header and add some instructions on how users can have your project up and running locally on their machine
- Add the "Acknowledgement" header and ad its content giving appropriate credit to Figma, for using their design template.
- Update the MIT.md file and add the licensed content as of now, the link is broken.
- Update the description of your portfolio to be more descriptive as what you have now doesn't say anything about your portfolio.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @AbigiyaTY ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions

- Kindly adjust the styling in your header so that your hamburger is well aligned to the left end.
- Kindly ensure the first section of your portfolio is well aligned in the center to improve your design.

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @AbigiyaTY,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**